Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -109,8 +109,12 @@ void Processor::initialize(u32)
         asm volatile("wfi");
 }
 
-void Processor::flush_tlb_local(VirtualAddress, size_t)
+void Processor::flush_tlb_local(VirtualAddress vaddr, size_t page_count)
 {
+    // NOTE: We must flush the data cache here, such that the updated page tables actually
```

Comment:
Unsolicited writing advice: always `s/note: //` as it's always information-free (why else would you write a comment).

(Optional nit, obvsly)

Reply:
(i go for `XXX` if something is really not obvious, but i vaguely remember this tag was some kind of surprise here?)

Reply:
> (i go for `XXX` if something is really not obvious, but i vaguely remember this tag was some kind of surprise here?)

We don't use `XXX`, as `NOTE` it's superfluous and mostly found in older code.

Reply:
Sure, will remove the `NOTE`, however in Andreas latest video he was adding such a `NOTE` comment too (later removed it tho), but that was in LibWeb, so 🤷